src/guiguts/file.py

```diff
@@ -7,7 +7,7 @@
 from tkinter import filedialog, messagebox, simpledialog
 from typing import Any, Callable, Final, TypedDict, Literal
 
-from guiguts.mainwindow import maintext, sound_bell
+from guiguts.mainwindow import maintext, sound_bell, logger
```

Comment:
What I've usually seen -- not saying this is correct -- is for the logger to be defined in every file, not imported from elsewhere. That way anything could freely log without needing to import other code besides `logging`.

Reply:
Thanks - all files that need access to the logger now use `getLogger` at the top of the file.